terraform/modules/content_delivery_network/lambda_functions/redirect_latest_tile_cache/src/lambda_function.py

```diff
@@ -53,15 +53,15 @@ def handler(event, context):
     dataset = path_items[1]
     if dataset == "cog":
         query_params = parse_qs(query_string)
-        dataset = query_params.get("dataset", [None])[0]
+        dataset = query_params.get("dataset", ["cog"])[0]
```

Comment:
Maybe you can put a comment here about why there is no dataset value in the cog case?  I think this is a temporary work-around or a workaround of a bug somewhere else, so can you put in the details of that bug here?